resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -381,4 +381,34 @@ object KubernetesUtils extends Logging {
       }
     }
   }
+
+  /**
+   * This function builds the EnvVar objects for each key-value env.
```

Comment:
The function description is inconsistent from the code body because ew don't build EnvVar when value is null.

Reply:
I think we should mention it, because it's supported to set on yaml and works normally in container.